webpack/scenes/ContentViews/Details/Filters/CVErrataIDFilterContent.js

```diff
@@ -239,8 +237,8 @@ const CVErrataIDFilterContent = ({
     dateType === 'issued' &&
     statusSelected === ALL_STATUSES;
 
-  const emptyContentTitle = __('No errata available to add to this filter.');
-  const emptyContentBody = __('No errata available for this content view.');
+  const emptyContentTitle = __('No errata filter rules yet');
+  const emptyContentBody = __('Add repositories with errata to content view to select them here.');
```

Comment:
I don't understand this. Is there a clearer way to word it?

Reply:
Updated to this:
```
const emptyContentTitle = __('No errata filter rules yet');
const emptyContentBody = __('No errata to add yet');
```